pip-requirements-doc

```diff
@@ -7,7 +7,6 @@ scikit-image
 pandas
 pytz
 beautifulsoup4
-ipython
```

Comment:
@saimn , why did you include `ipython` in #7711 ? I don't see why it is needed for doc build. Do you remember?

Reply:
I don't know if it is needed, but it was in the conda environment (https://github.com/astropy/astropy/pull/7711/files#diff-f6547eedc6811e1dd247024c3940b0c3L24).

Reply:
we definitely need it for the tests (https://github.com/astropy/astropy/blob/51c619e5ffe7aaf748e7c34d08c6336762decc74/astropy/table/tests/test_jsviewer.py#L159)